packages/unit-vitest/package.json

```diff
@@ -39,18 +39,18 @@
   },
   "devDependencies": {
     "@types/lodash-es": "^4.17.12",
-    "@vitest/ui": "^2.0.5",
-    "@vue/test-utils": "^2.4.4",
+    "@vitest/ui": ">=1.6.1 <2.0.0 || >=2.1.9 <3.0.0 || >=3.0.5",
+    "@vue/test-utils": "^2.0.5",
     "quasar": "^2.14.6",
     "rimraf": "^5.0.5",
-    "vitest": "^2.0.5",
+    "vitest": ">=1.6.1 <2.0.0 || >=2.1.9 <3.0.0 || >=3.0.5",
     "vue": "^3.4.21"
   },
   "peerDependencies": {
     "@vitest/ui": "^1.1.0 || ^2.0.0",
     "@vue/test-utils": "^2.4.3",
     "quasar": "^2.12.7",
-    "vitest": "^1.1.0 || ^2.0.0",
+    "vitest": ">=1.6.1 <2.0.0 || >=2.1.9 <3.0.0 || >=3.0.5",
```

Comment:
```suggestion
    "vitest": "^1.1.0 || ^2.0.0 || ^3.0.0",
```

Reply:
This change was actually partly legitimate, it was meant to prevent users to use versions of Vitest which are affected by [this CVE](https://github.com/vitest-dev/vitest/security/advisories/GHSA-9crc-q9x8-hgqq)

Reply:
I just don't think we are the "authority" to enforce this :) A security warning is more prominent than a peer dependency warning anyway. So, it ends up complicating our version ranges. But, I am fine if you are fine with it.

Reply:
Well, we are the authority when it comes to quasar testing packages
This has been done already with other packages over which we provide a thin layer/ease the integration

Also, [the actual fix](https://github.com/quasarframework/quasar-testing/blob/6734d138719115dc6fa190f9898fab8614c9f209/packages/unit-vitest/package.json#L49) is way simpler than the one proposed by Calson, which was just a copy/paste from the CVE page